pocs/scheduler/constraint.py

```diff
@@ -159,3 +120,40 @@ def get_score(self, time, observer, observation, **kwargs):
 
     def __str__(self):
         return "Moon Avoidance"
+
+
+class Altitude(BaseConstraint):
+
+    """ Implements altitude constraints for a horizon """
+
+    def __init__(self, horizon=None, *args, **kwargs):
+        """
+        Retrieves the coordinate list from the config file and validates it
```

Comment:
Doesn't appear to retrieve anything.

Reply:
Oops, forgot to update docstring, thanks.